deploy/terraform/main.tf

```diff
@@ -0,0 +1,135 @@
+##################################################################################
+# PROVIDERS
+##################################################################################
+
+# NOTE before others start working on the TF files, we need to setup
+# remote backend. TF does not recommend checking in tfstate  and lock file
+# into git so we must configure a S3 bucket that will host the tfstate and lock file
+# that way we all pull the same state file when someone makes changes to TF.
+
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "~> 4.53.0"
+    }
+  }
+}
+
+provider "aws" {
+  profile = var.aws_profile
+}
+##################################################################################
+# DATA
+##################################################################################
+
+data "aws_subnet" "subnet_1" {
+  id = var.subnet_id
+}
+
+data "aws_security_group" "default_sg" {
+  id = var.default_sg
+}
+
+data "aws_ssm_parameter" "ami" {
+  name = "/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2"
+}
+
+##################################################################################
+# RESOURCES
+##################################################################################
+
+########### Container service ##########
+resource "aws_ecs_cluster_capacity_providers" "osmt_container_cluster" {
+  cluster_name = "osmt-dev-cluster"
```

Comment:
Is there a reasonable way to parameterize the stack name? This could also be used for tags.